thehylia.py

```diff
@@ -488,8 +488,8 @@ def doIt(): # Only in a function to be able to stop after errors, really.
             if onlySearch:
                 searchResults = search(searchTerm)
                 if searchResults:
-                    print("Soundtracks found (to download, "
-                          "run \"{} soundtrack-name\"):".format(SCRIPT_NAME))
+                    #  print("Soundtracks found (to download, "
+                            #  "run \"{} soundtrack-name\"):".format(SCRIPT_NAME))
```

Comment:
I removed this, so it works better with my scripts. It can be still kept if the scripts are modified to take it consideration but I decided to not bother :stuck_out_tongue: